src/data/weapons.py

```diff
@@ -204,4 +204,61 @@
         "type": " sword",
         "size": 9
     }
-}
\ No newline at end of file
+}
+
+Range_Weapon_Data = {
+	"heavy_bow": {
+		"mat_types": ["metal"],
+		"notable": "BOWlder",
```

Comment:
Need a better name, have a look at the names I have given to the melee weapons :)